recipes/wxmpl/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "wxmpl" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: wxmpl-v{{ version }}.tar.gz
+  url: https://github.com/NOAA-ORR-ERD/wxmpl/archive/v{{ version }}.tar.gz
+  sha256: 20258f22b0affbd56968af8b0743e99bf8ade4845cbb4ce057398cd693787c7e
+
+build:
+  number: 0
+  noarch: python
+#  script: pip install --no-deps --no-cache-dir ./
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python <=3.10
```

Comment:
Readme says `>=3.8`, setup.py metadata says `=2.7`.

Reply:
Honestly, I haven't tested it on very many versions lately -- but I'll get it all in sync.